fe/fe-core/src/main/java/org/apache/doris/cloud/datasource/CloudInternalCatalog.java

```diff
@@ -145,11 +148,19 @@ protected Partition createPartitionWithIndices(long dbId, OlapTable tbl, long pa
                         tbl.getEnableUniqueKeyMergeOnWrite(), tbl.storeRowColumn(), indexMeta.getSchemaVersion());
                 requestBuilder.addTabletMetas(builder);
             }
+            if (!storageVaultIdSet) {
+                requestBuilder.setStorageVaultName(storageVaultName);
+            }
 
             LOG.info("create tablets, dbId: {}, tableId: {}, tableName: {}, partitionId: {}, partitionName: {}, "
                     + "indexId: {}",
                     dbId, tbl.getId(), tbl.getName(), partitionId, partitionName, indexId);
-            sendCreateTabletsRpc(requestBuilder);
+            // dataProperty.setStorageVaultId(storageVaultId);
```

Comment:
we have to check the existence when creating a table with storage vault,
it seems that check is missing?

Reply:
Currently the existence check logic lies in MS's code, so the rpc procedure already overlaps the check logic.